common/service/config/config.go

```diff
@@ -232,6 +232,8 @@ type (
 		Level string `yaml:"level"`
 		// OutputFile is the path to the log output file
 		OutputFile string `yaml:"outputFile"`
+		// levelKey is the desired log level
+		LevelKey string `yaml:"levelKey"`
```

Comment:
maybe:
` LevelKey string `default:"level" yaml:"levelKey"``

Reply:
I have tried that before, it doesn't work with the yaml library that we are using. (otherwise we have done this for lots of other fields)

LMK if I am wrong.

Reply:
I've tried with --> LevelKey string `default:"mylevel" yaml:"levelKey"`
but it doesn't work for me either